src/engine/controls/cuecontrol.h

```diff
@@ -50,8 +51,8 @@ class HotcueControl : public QObject {
     void setCue(CuePointer pCue);
     void resetCue();
     void setPosition(double position);
-    void setColor(PredefinedColorPointer newColor);
-    PredefinedColorPointer getColor() const;
+    void setColor(const mixxx::RgbColor::optional_t& newColor);
```

Comment:
Always pass RgbColor (+ optional) by-value. This is type is small and trivially copyable.